src/Common/tests/System/Threading/Tasks/TaskTimeoutExtensions.cs

```diff
@@ -13,10 +13,10 @@ public static class TaskTimeoutExtensions
         {
             var cts = new CancellationTokenSource();
 
-            if (task == await Task.WhenAny(task, Task.Delay(millisecondsTimeout, cts.Token)))
+            if (task == await Task.WhenAny(task, Task.Delay(millisecondsTimeout, cts.Token)).ConfigureAwait(false))
             {
                 cts.Cancel();
-                await task;
+                await task.ConfigureAwait(false);
```

Comment:
This one doesn't actually need ConfigureAwait(false), as we know by construction that the task has already completed here, and this await is just to propagate any exceptions that may have occurred.  But it's also fine to leave it.